### PR TITLE
Cypress/e2e/fixed cypress test cases

### PR DESCRIPTION
**Pull request by furqanmlk**
#### Summary
fixed cypress test cases
#### Screenshots
image

### CODE REVIEW

File: e2e/cypress/support/ui/team.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 Cypress.Commands.add('uiInviteMemberToCurrentTeam', (username) => {
     // # Open member invite screen
-    cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+    cy.get('.icon-chevron-down').should('be.visible').click();
     cy.get('#invitePeople > button').should('contain.text', 'Invite People').click();
```
Comment (saturninoabril) [inline e2e/cypress/support/ui/team.js:7]: I'd recommend using `cy.uiOpenTeamMenu('Invite People');`

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @furqanmlk! LGTM, except for one request.

**Comment by furqanmlk** [comment]
image

**Review by saturninoabril — APPROVED** [review]
Thanks @furqanmlk!

**Review by josephbaylon — APPROVED** [review]
LGTM

**Pull request merged**